packages/app/babel.config.js

```diff
@@ -22,4 +22,5 @@ module.exports = {
       ],
     },
   },
+  plugins: ['@babel/plugin-proposal-optional-chaining'],
```

Comment:
This is for making tests work?

Reply:
It's for making the `build` script work

Reply:
Hmm, we don't use this file in `app`, that's what confuses me. We only use this file for running tests. The other babel configs (`babel.dev.js` and `babel.prod.js`) already have the optional chaining libs.

Reply:
Running the `build` script sets the `env` to `production`, so I can't include it in `test` environment

https://github.com/codesandbox/codesandbox-client/blob/a98b774783bd183539996f181b3ec5ed00ced84e/packages/app/package.json#L12

Reply:
So removing this entry breaks the prod build? I thought we used optional chaining already, this babel config is not used in prod.

Reply:
Yeah this breaks build indeed.

We only use optional chaining in `common`, not in `app` atm.

Reply:
Ah, right, I also did this in the `follow-templates` branch. It would be best to add the plugin to `babel.dev.js` and `babel.prod.js`, as we set all plugins there.

Reply:
Moved the plugin to `babel.dev.js` & `babel.prod.js`